Makefile

```diff
@@ -178,6 +178,10 @@ config: $(LUA) FORCE
 	$(GLUON_ENV) $(LUA) scripts/target_config_check.lua
 
 
+container: FORCE
+	@scripts/container.sh
```

Comment:
Does it really make sense to put this in the Makefile, even though it is not part of the actual build?

I'd just put the script under `contrib/` to be called without the Makefile.

Reply:
I have no particular love for `make container` besides establishing good discoverability.

Setting up a build environment is likely as much part of the actual build as the linter steps.